contributing.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to the CEPC and the Positive Work Environment Community Group
+
+We welcome contributions from the community as we work to develop a positive and inclusive work environment for W3C. 
+
+If you have feedback or questions relating to the Code of Ethics and Professional Conduct or any of our other projects, please log an issue in this repository. Logging issues allows the chairs to review and raise issues in upcoming meetings. 
+
+We ask that participants do not open Pull Requests with changes to the CEPC or other documents unless asked to do so by the chairs or editors. This is to help us keep discussions in issues or meetings where appropriate, and to help prevent merge conflicts. 
```

Comment:
There are probably better ways to deal with merge conflicts, such as asking people to rebase their pull requests if there are conflicts, but I think the approach to discussions is more important here?

Reply:
I personally (as an Editor) prefer that people start a discussion in issues - "I see this problem, what would you think of a solution like this?" before going through the work to file a PR, UNLESS it is a typo-level change.  (Like, REALLY a typo, not "I don't like this wording".). It's not strictly "unless asked to do so by the chairs", but it avoids the stigma of me (as editor) "rejecting" pull requests because they weren't consensus, or were deemed unnecessary, or were wrapped up in larger changes in flight.

Reply:
I suggest turning these points into a bullet list, and adding the following two bullets:

- Please be as specific as possible, both in describing the issue and (ideally) in suggesting a remedy.

- If you have multiple substantive issues to submit, please consider submitting them one at a time, waiting for each issue to be resolved by the group before opening the next one.  This will help keep the group's attention focused and help to avoid overwhelming the chairs and other members who typically have day jobs and other responsibilities.